.github/onboarding.md

```diff
@@ -0,0 +1 @@
+👷 **__We are currently working on an improved onboarding experience!__** 👷
```

Comment:
Short Answer:
To me, it seems to be the logically correct position based on the location of the contribution.md file.

Long Answer:
As discussed with the coordinators of all teams, the plan is to automate the onboarding process. For this purpose, we agreed to use markdown files on Github. The core of the onboarding process will be introducing the workflow at Catrobat's development teams. This workflow is already (partly) described in the contribution.md file at Catroweb and will be adapted to fit the needs of newcomers and current contributors. It will serve as a step-by-step guide through the development process of a single ticket. Additional information that is only relevant to the newcomers will be put into the onboarding.md file to serve as a source of information linking to existing documentation (e.g., wiki, confluence) and the contribution.md file. The contribution.md file is correctly placed in the .github, and used in https://github.com/Catrobat/Catroweb/community. Creating a documentation directory for a single file (onboarding.md) is a bit overhead in this case.